scaffold-tfmigrate/action.yaml

```diff
@@ -32,6 +32,15 @@ runs:
           sed "s|%%TARGET%%|${TFACTION_TARGET}|g" ${{ github.action_path }}/tfmigrate.hcl |
           sed 's|%%S3_BUCKET_NAME_TFMIGRATE_HISTORY%%|${{steps.target-config.outputs.s3_bucket_name_tfmigrate_history}}|g' > .tfmigrate.hcl
         fi
+      if: steps.target-config.outputs.s3_bucket_name_tfmigrate_history != ''
+      shell: bash
+      working-directory: ${{steps.target-config.outputs.working_directory}}
+    - run: |
+        if [ ! -f ".tfmigrate.hcl" ]; then
+          sed "s|%%TARGET%%|${TFACTION_TARGET}|g" ${{ github.action_path }}/tfmigrate-gcs.hcl |
```

Comment:
This file name is wrong. Not tfmigrate-gcs.hcl but tfmigrate-gcp.hcl.

But I prefer tfmigrate-gcs.hcl than tfmigrate-gcp.hcl,
because the backend is not GCP but GCS.

Reply:
Nice catch! I've renamed them by 697ec09. Thanks!